src/Components/Seo/StructuredData.tsx

```diff
@@ -1,13 +1,13 @@
 import { Meta } from "react-head"
-import type { Thing, WithContext } from "schema-dts"
+import type { Graph, Thing, WithContext } from "schema-dts"
 
-interface StructuredDataProps<T extends Thing> {
-  schemaData: WithContext<T>
+type SchemaData = WithContext<Thing> | Graph
+
+interface StructuredDataProps {
+  schemaData: SchemaData
```

Comment:
Main issue here. The way to figure this out would be to just import the type directly and see what the problem was. The issue wound up just being that Graph is itself a top level type (sans `WithContext`)